modules/test/playwright/tests/object-web/objectFields.spec.ts

```diff
@@ -56,7 +56,68 @@ test.describe('Manage object fields through Model Builder', () => {
 		);
 
 		await apiHelpers.listTypeAdmin.deleteListTypeDefinition(
-			ListTypeDefinition.id
+			listTypeDefinition.id
 		);
 	});
+
+	test('all picklist definitions are listed during object field creation', async ({
+		apiHelpers,
+		modelBuilderPage,
+		page,
+		viewObjectDefinitionsPage,
+	}) => {
+		const listTypeDefinitions = await Promise.all(
+			Array(22)
+				.fill(null)
+				.map(() =>
+					apiHelpers.listTypeAdmin.postRandomListTypeDefinition()
+				)
+		);
+
+		const objectDefinition =
+			await apiHelpers.objectAdmin.postRandomObjectDefinition('default');
+
+		try {
+			await page.goto('/');
+
+			await viewObjectDefinitionsPage.goto();
+
+			await viewObjectDefinitionsPage.openObjectFolder('default');
+
+			await viewObjectDefinitionsPage.viewInModelBuilder();
+
+			await modelBuilderPage.openObjectFieldSelectionPage(
+				objectDefinition.name,
+				'Picklist',
+				'objectFieldLabel' + getRandomInt()
+			);
+
```

Comment:
You might need to add extra steps here because of reducing the scope of  `openObjectFieldSelectionPage()`.